Description: add a required mojom file that's missing from the source tarballs
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- /dev/null
+++ b/chrome/test/data/webui/web_ui_test.mojom
@@ -0,0 +1,9 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+module web_ui_test.mojom;
+interface TestRunner {
+  // Called by the test when it finishes. Indicates failure via a non-empty
+  // |failureMessage|.
+  TestComplete(string? failureMessage);
+};
